pyirf/benchmarks/tests/test_angular_resolution.py

```diff
@@ -45,11 +51,22 @@ def test_angular_resolution(unit):
     events["true_energy"].value[500] = np.nan
     events["true_energy"].value[1500] = np.nan
 
-    ang_res = angular_resolution(
-        events,
-        [1, 10, 100] * u.TeV,
-    )['angular_resolution']
+    bins = [1, 10, 100] * u.TeV
 
+    table = angular_resolution(events, bins)
+    ang_res = table['angular_resolution'].to(u.deg)
     assert len(ang_res) == 2
     assert u.isclose(ang_res[0], TRUE_RES_1 * u.deg, rtol=0.05)
     assert u.isclose(ang_res[1], TRUE_RES_2 * u.deg, rtol=0.05)
+
+    # 2 sigma coverage interval
+    quantile = norm(0, 1).cdf(2) - norm(0, 1).cdf(-2)
```

Comment:
Maybe add a utility function for this? The user will need to compute this anyways if they want to compute anything but the 1sigma angular resolution, so we might as well provide it. Its ofc. a one-liner though. Could be reused for the default value in `angular_resolution.py`

Reply:
common values are stuff like 68%, 90 %, or 95%. 90 is not close to any integer multiple of standard deviations...

This was more to have  a good test case.